File: Lab3/app/src/main/java/com/example/lab3/MainActivity.java
package com.example.lab3;

import androidx.appcompat.app.AppCompatActivity;

import android.content.res.Configuration;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    private String getScreenOrientation(){
        if(getResources().getConfiguration().orientation ==
                Configuration.ORIENTATION_PORTRAIT)
            return "Портретна орієнтація";
        else if (getResources().getConfiguration().orientation ==
                Configuration.ORIENTATION_LANDSCAPE)
            return "Альбомна орієнтація";
        else
            return "";
    }
    public void onClick1(View v) {
        Toast.makeText(this, getScreenOrientation(), Toast.LENGTH_SHORT).show();
    }
    }

File: Lab1/app/src/main/java/com/example/lab1/MainActivity.kt
package com.example.lab1

import android.content.Intent
import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
        fun send (view: View) {
            if (editText.text.toString() == "" || editText2.text.toString() == "") {
                textView4.setTextColor(Color.parseColor("#ED0909"))
                val handler = android.os.Handler()

                handler.postDelayed({ textView4.setTextColor(Color.parseColor("#14778F")) }, 1600)
            }
            else {
                val inf = Intent(this, Act2::class.java)
                val NameStr = editText.text.toString()
                var GroupStr = editText2.text.toString()
                inf.putExtra(Act2.NAME, NameStr)
                inf.putExtra(Act2.GROUP, GroupStr)
                startActivity(inf)
            }
        }

    fun info (view: View) {
            val inf = Intent(this, Act2::class.java)
            startActivity(inf)
    }
}
